compiler/daml-lf-proto/src/DA/Daml/LF/Proto3/Decode.hs

```diff
@@ -6,7 +6,7 @@ module DA.Daml.LF.Proto3.Decode
   , decodePayload
   ) where
 
-import Da.DamlLf (ArchivePayload(..), ArchivePayloadSum(..))
+import Com.Digitalasset.DamlLf.DamlLf (ArchivePayload(..), ArchivePayloadSum(..))
```

Comment:
I guess the reason why this changes for Haskell but not for Java/Scala is due to the `java_package` annotation?

Reply:
Indeed. As far I can tell haskell proto3-suite do not provide the equivalent.